web/src/queries/l10n.ts

```diff
@@ -123,26 +123,22 @@ const useL10nConfigChanges = () => {
 
 /// Returns the l10n data.
 const useL10n = () => {
-  const [
-    { data: config },
-    { data: locales },
-    { data: keymaps },
-    { data: timezones }
-  ] = useSuspenseQueries({
-    queries: [
-      configQuery(),
-      localesQuery(),
-      keymapsQuery(),
-      timezonesQuery()
-    ]
-  });
+  const [{ data: config }, { data: locales }, { data: keymaps }, { data: timezones }] =
+    useSuspenseQueries({
+      queries: [configQuery(), localesQuery(), keymapsQuery(), timezonesQuery()]
+    });
```

Comment:
Ouch, I didn't realize I sent these format changes made by Zed editor which I'm testing. Will revert them in a new commit.

Reply:
After a short discussion with @imobachgs, I will keep it as it is because we're gonna format all the code base for consistence.

Reply:
BTW, the format change was not because Zed but because Zed applying the rules we have at .prettierrc file

Reply:
> we're gonna format all the code base for consistence.

Done at https://github.com/openSUSE/agama/pull/1460